### DISCUSSION TITLE
Hide website preview in tweet

### DISCUSSION THREAD

**Discussion by fabston** [tweepy/tweepy#1946] [Questions] [answered]
Hi there,

Is it possible to hide the website previews in a tweet? It's possible if doing manually as shown below:

![tweet](https://i.imgur.com/hNTskwX.png)

**Comment by Harmon758** [comment, marked as answer]
This isn't something that's officially supported with Twitter's API.

There's [an undocumented method that you can try](https://stackoverflow.com/a/65550091/6191255).
